unit-tests/func/hw-errors/l500/l500-error-common.h

```diff
@@ -0,0 +1,95 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../../../test.h"
+#include <concurrency.h>
+
+using namespace rs2;
+
+const uint8_t l500_trigger_error_opcode = 0x5E;
+
+// Possible error values
+const std::map< uint8_t, std::string > l500_error_report
+    = { { 1, "Fatal error occur and device is unable \nto run depth stream" },
+        { 2, "Overflow occur on infrared stream" },
+        { 3, "Overflow occur on depth stream" },
+        { 4, "Overflow occur on confidence stream" },
+        { 5, "Receiver light saturation, stream stopped for 1 sec" },
+        { 6, "Error that may be overcome in few sec. \nStream stoped. May be recoverable" },
+        { 7, "Warning, temperature close to critical" },
+        { 8, "Critical temperature reached" },
+        { 9, "DFU error" },
+        { 12, "Fall detected stream stopped" },
```

Comment:
Where're 10, 11? If it's meant to be like this, please explain

Reply:
Same with 13?

Reply:
added